torch/__init__.py

```diff
@@ -521,65 +521,130 @@ def is_warn_always_enabled():
 ################################################################################
 
 from ._tensor import Tensor
-from .storage import _StorageBase
+from .storage import _StorageBase, TypedStorage
 
-
-class DoubleStorage(_C.DoubleStorageBase, _StorageBase):
+class ByteStorage(_C.ByteStorageBase, _StorageBase):
     pass
 
+class DoubleStorage(TypedStorage):
+    @classproperty
+    def dtype(self):
+        return torch.double
 
-class FloatStorage(_C.FloatStorageBase, _StorageBase):
-    pass
+    def __init__(self, *args, **kwargs):
```

Comment:
why do you need to override constructor in this way? Seems like it can be just omitted.

Reply:
You're right. I think at some point these did something, but not anymore